components/create_request/create_request_component.js

```diff
@@ -36,35 +38,42 @@ export default class CreateRequestComponent extends React.Component {
         </Header>
         <Content padder>
           <Form>
+            <Item floatingLabel>
+              <Label>Post Title</Label>
+              <TextBox padder
+                     onChangeText={(input) => this.setState({postTitle: input})} />
```

Comment:
You can make a generic `handleInputChange` for all of your form events, the React docs go over this pretty well: https://reactjs.org/docs/forms.html#handling-multiple-inputs